org.adempiere.base/src/org/compiere/model/MSequence.java

```diff
@@ -542,8 +569,7 @@ public static String getDocumentNoFromSeq(MSequence seq, String trxName, PO po)
 
 		//	create DocumentNo
 		StringBuilder doc = new StringBuilder();
-		if (prefix != null && prefix.length() > 0) {
-			String prefixValue = Env.parseVariable(prefix, po, trxName, false);
+		if (prefixValue != null && prefixValue.length() > 0) {
 			if (!Util.isEmpty(prefixValue))
```

Comment:
shouldn't use double if here, just change "!Util.isEmpty(prefixValue)" to "!Util.isEmpty(prefixValue, true)